pigeon/weights/templates/registration/register.html

```diff
@@ -6,11 +6,12 @@
   <h1 class="text-center col-lg-12 pigeon-font pigeon-color margin-bottom-20">{% trans "Register" %}</h1>
   <form method="post" autocomplete="off" class="form-contact col-lg-offset-2">
     {% csrf_token %}
-    {% for field in user_form %}
-      {% bootstrap_field field layout="horizontal" form_group_class="col-md-12" field_class="col-md-9 input-form-space" show_help=False show_label=False %}
-    {% endfor %}
-    {% for field in profile_form %}
-      {% bootstrap_field field layout="horizontal" form_group_class="col-md-12" field_class="col-md-8 input-form-space" label_class="register-label col-md-1" show_help=False %}
+    {% for field in register_form %}
+      {% if field.name in 'language,country,avatar'|slice:',' %}
```

Comment:
No way to just do `in ['language', 'country', 'avatar']`?

Reply:
Nope, thanks to the wonderful template language of django. This solution is a well known quick and dirty workaround. A cleaner solution could be to define the list in the view and use the variable in template but it is less understandable imo.